### ISSUE TITLE
RC Spot Check: Molecules and Light 1.4.0-rc.2

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#89]
After #84, Molecules and Light 1.4.0-rc.2 is ready for a spot check.

[Link to sim](https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-rc.2/molecules-and-light_en.html)

[Link to iFrame](https://www.colorado.edu/physics/phet/dev/html/molecules-and-light/1.4.0-rc.2/molecules-and-light_en-iframe.html)

**Issues to Verify**
Please test the following issues and check them off after addressing. If they are resolved, close them. If not, they should be updated.
- [ ] https://github.com/phetsims/molecules-and-light/issues/175

Changes for the above are targeted for Edge, so please verify that all is well on that platform. But for sanity please also spot check on a few platforms.

If any new issues are found, please note them in https://github.com/phetsims/molecules-and-light/issues and reference this issue.

@ariel-phet can you please prioritize?

**Comment by jessegreenberg** [comment]
As mentioned in https://github.com/phetsims/molecules-and-light/issues/175, @phet-steele noticed that performance in Edge still tanks when hovering over other buttons in the UI. I tried rendering the whole sim with SVG and performance really improves dramatically. I don't see any strange rendering artifacts that have been reported in other sims. I am going to prepare a new RC with this Edge specific change.

**Comment by jessegreenberg** [comment]
We also want to look into https://github.com/phetsims/perennial/issues/79 before the next Rc
